### ISSUE TITLE
:espanso wouldn't work in the testing expansion view

### ISSUE THREAD

**Issue by MikhailTNY** [federico-terzi/espanso#872]
Hi, 

On W11 with v2.1.0-alpha, if you're doing an automated migration and then it asks to test `:espanso`, it doesn't work. 

It is possible this is because it wouldn't be in the migrated files if this was included in the new snippet file.

**Comment by federico-terzi** [comment]
Hey @MikhailTNY,

Thanks for reporting this issue! Interesting, I didn't think about this edge case

> It is possible this is because it wouldn't be in the migrated files if this was included in the new snippet file.

Just to make sure we are on the same page, did you remove ":espanso" from your snippets?

Cheers :)

**Comment by MikhailTNY** [comment]
> Just to make sure we are on the same page, did you remove ":espanso" from your snippets?

I removed it a long time ago, so yes.

**Comment by tiktuk** [comment]
Same, this also confused me, I thought Espanso wasn't working because of this. Also removed it a long time ago.

**Comment by federico-terzi** [comment]
Makes sense, thank you, guys! I guess we'll have to create an ad-hoc match for that welcome screen
